### ISSUE TITLE
Retwaats appearing as replies

### ISSUE THREAD

**Issue by KibbeWater** [KibbeWater/Twatter#11]
# NOTES:
* Retwaat does not contain the quoted Retwaat
* Becomes reply of retwaated tweet

Network Result:
```json
{
    "success": true,
    "data": {
        "_id": "64a31a48f817bbf3fc5c747d",
        "user": {
            ...
        },
        "content": "Test...",
        "images": [],
        "videos": [],
        "parent": "64914c9b221bfb6cd5b35d69", // Marked as Reply
        "comments": [],
        "likes": [],
        "retwaats": [],
        "mentions": [],
        "date": 1688410696593,
        "__v": 0
    }
}
```

**Comment by KibbeWater** [comment]
Issue Located:

https://github.com/KibbeWater/Twatter/blob/8a5e492dcb8ebed2d0511ba25207a7617f69b0f1/components/Modals/PostModal.tsx#L30

**Comment by KibbeWater** [comment]
Fixed in #13
